### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by austino1976**
Fix for CR-1050591

### CODE REVIEW

**Comment by rradjabi** [comment]
Retest this please

**Comment by aye20** [comment]
> Does aws F1 have ENV(XILINX_VITIS) set?
> I mean, when build on aws, do we make sure the ert is loaded into the emu.so?

We look for the XILINX_VITIS if it exists, the libsched_em.so is added, if not libsched_em.so does not get built.

**Comment by maxzhen** [comment]
ok to test

**Comment by xuhz** [comment]
we have to make sure XILINX_VITIS is set on aws -- that is what this fix is doing, right? 
on aws, we can't access /tools/..., then the libsched_em.so doesn't get built, and then the emu doesn't work. 
what this fix is doing is to make sure libsched_em.so is built. so XILINX_VITIS has to be set on aws

File: src/runtime_src/ert/CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-if ((EXISTS /tools/batonroot/rodin/devkits) AND (${XRT_NATIVE_BUILD} STREQUAL "yes") AND (${CMAKE_HOST_SYSTEM_PROCESSOR} STREQUAL "x86_64"))
+if ((EXISTS $ENV{XILINX_VITIS}/gnu/microblaze/lin) AND (${XRT_NATIVE_BUILD} STREQUAL "yes") AND (${CMAKE_HOST_SYSTEM_PROCESSOR} STREQUAL "x86_64"))
```
Comment (stsoe) [inline src/runtime_src/ert/CMakeLists.txt]: Where is XILINX_VITIS set?

File: src/runtime_src/ert/ert.mk [hunk 2]
```diff
@@ -1,5 +1,4 @@
-RDI_DEVKITS_HOME := /tools/batonroot/rodin/devkits
-MB_HOME := $(RDI_DEVKITS_HOME)/lnx64/cross/RHEL6/microblaze/lin
+MB_HOME := ${XILINX_VITIS}/gnu/microblaze/lin
```
Comment (stsoe) [inline src/runtime_src/ert/ert.mk]: Are we sure this is the same compiler?  This is a pretty scary change unless we are certain the compiler to use.

Comment (stsoe) [reply src/runtime_src/ert/ert.mk]: Even if I set XILINX_VITIS and use the MB_HOME, the sched.bin file are not identical to before.

While I agree with the intent of this PR and that using XILINX_VITIS is better that using the internal mb compiler,  making this change in 2019.2 branch is not possible.      

Please redo this PR and make it specific to what you are after, such that it does not affect building of the microblaze firmware used in hardware.

**Comment by austino1976** [comment]
Superseded by PR#2617

**Pull request closed without merging**